frontend/src/screens/study-landing.tsx

```diff
@@ -48,6 +48,7 @@ const CompletedMessage:React.FC<{
                 <h5 css={{ lineHeight: '150%', marginBottom: '3rem' }}>
                     You‘ve completed a Kinetic activity.
                     {!aborted && ' This task will be marked as complete on your dashboard.'}
+                    {!consented && `Since you have chosen to not allow us to use your results in our research, you did not earn ${<Points study={study} /> }. You can retake the study and change that, if you wish` }
                 </h5>
```

Comment:
have we ran this verbiage by Debshila or Vicky?

Reply:
Nope, I threw something in, figured it's be resolved in P.R. :)